### PR TITLE
UML-974 Translation switch - DO NOT MERGE YET

### PR DESCRIPTION
**Pull request by GemTay**
# Purpose

Adds a UI switch to enable users to switch between english and welsh

Fixes UML-974

## Approach

Created a common partial twig template to match the prototype. The switch checks whether the current url contains the welsh prefix, and if so displays "Cymraeg" as text rather than a link - and vice versa for the english link. It adds a /cy prefix to the welsh link and removes it from the english link

## Checklist

* [x] I have performed a self-review of my own code
* [ ] I have added relevant logging with appropriate levels to my code
* [ ] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
* [x] I have added tests to prove my work
* [ ] The product team have tested these changes

### CODE REVIEW

File: service-front/app/src/Common/templates/partials/welsh-switch.html.twig [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+        <ul class="trans-switch">
+            <li class="govuk-list trans-switch__list govuk-body-s">
+                {% if path() matches '/(\\/cy.*)/' %}
```
Comment (cooperaj) [inline service-front/app/src/Common/templates/partials/welsh-switch.html.twig:5]:
```suggestion
                {% if path() matches '/(^\\/cy.*)/' %}
```
Might need this or something like /lpa/cymraeg.html will match.

Comment (GemTay) [reply service-front/app/src/Common/templates/partials/welsh-switch.html.twig]: Good spot - fixed :)

**Review by Raffers — DISMISSED** [review]

**Comment by williamfalconeruk** [comment]
I thin this would have been better to be put into a base layout page, rather than as an include on each of the pages in the service. Or this might just be my understanding of twig...

**Review by williamfalconeruk — DISMISSED** [review]

**Review by SeemaMenon — APPROVED** [review]
LGTM

**Comment by SeemaMenon** [comment]
> I think this would have been better to be put into a base layout page, rather than as an include on each of the pages in the service. Or this might just be my understanding of twig...

I too did think the same actually !

**Review by cooperaj — APPROVED** [review]
LGTM

**Review by SeemaMenon — APPROVED** [review]

**Pull request merged**